src/test/java/uk/nhs/digital/nhsconnect/lab/results/translator/mapper/BundleMapperTest.java

```diff
@@ -129,6 +154,30 @@ void testMapPathologyRecordToBundleWithPerformer() {
         );
     }
 
+    @Test
+    void testMapPathologyRecordToBundleWithPerformingOrganization() {
+        final var mockPerformingOrganization = mock(Organization.class);
+        pathologyRecordBuilder.requestingOrganization(mockPerformingOrganization);
```

Comment:
Could we use the `.performingOrganization` field from the builder?
```suggestion
        pathologyRecordBuilder.performingOrganization(mockPerformingOrganization);
```

Reply:
This is another copy+paste issue from my last test sorry. These tests look so similar it's easy to miss out 😛